infrastructure/code/convert.go

```diff
@@ -366,7 +367,7 @@ func (s *SarifResponse) toIssues() (issues []snyk.Issue) {
 	return issues
 }
 
-func getIssueId(ruleId string, path string, startLine int, endLine int, startCol int, endCol int) string {
+func getIssueKey(ruleId string, path string, startLine int, endLine int, startCol int, endCol int) string {
 	// deepcode ignore InsecureHash: The hash isn't used for security purposes.
 	id := md5.Sum([]byte(ruleId + path + strconv.Itoa(startLine) + strconv.Itoa(endLine) + strconv.Itoa(startCol) + strconv.Itoa(endCol)))
```

Comment:
Using md5 is still a smell :). I don't see any reason to not use a secure hash like sha-256, as performance is comparable.